app/models/miq_ui_worker.rb

```diff
@@ -20,4 +20,17 @@ def friendly_name
   end
 
   include MiqWebServerWorkerMixin
+  include MiqWorker::ServiceWorker
+
+  def self.supports_container?
+    true
+  end
+
+  def container_port
+    3001
```

Comment:
Should this be 3000?

Reply:
Nope, 3000 is exposed from the container and since the UI needs to serve assets as well as run the worker httpd listens on 3000 then points to 3001 for the worker.